aip/general/0202.md

```diff
@@ -81,6 +81,39 @@ backwards compatible.
 Any new `FieldInfo.Format` value **must** be governed by an
 [IETF-approved RFC][ietf rfc] or a [Google-approved AIP](./0001.md).
 
+### Type References
+
+Some generic field types (AIP-146), like `google.protobuf.Any`, can still have
```

Comment:
Link for AIP-146?

Reply:
So it *should* auto-link via the site-generator. [Here is the code](https://github.com/aip-dev/site-generator/blob/master/aip_site/models/aip.py#L59-L72).

Reply:
Cool :) I normally copy a link from elsewhere... we've got a lot of manually-linked ones. Maybe we should clean up at some point.

Reply:
Yeah I only recently double checked this and I wish it was more obvious because it's cool :)